helix-term/src/keymap.rs

```diff
@@ -350,6 +350,39 @@ impl Keymap {
         }
     }
 
+    pub fn reverse_map(&self) -> HashMap<String, Vec<Vec<KeyEvent>>> {
+        // recursively visit all nodes in keymap
+        fn map_node(
+            cmd_map: &mut HashMap<String, Vec<Vec<KeyEvent>>>,
+            node: &KeyTrie,
+            keys: &mut Vec<KeyEvent>,
+        ) {
+            match node {
+                KeyTrie::Leaf(cmd) => match cmd {
+                    MappableCommand::Typable { name, .. } => {
+                        cmd_map.entry(name.into()).or_default().push(keys.clone())
+                    }
+                    MappableCommand::Static { name, .. } => cmd_map
+                        .entry(name.to_string())
+                        .or_default()
+                        .push(keys.clone()),
+                },
+                KeyTrie::Sequence(_) => todo!(),
```

Comment:
This `todo!()` makes the editor panic if there's a sequence keybind in the user config.

Reply:
Sorry, totally missed it. I removed it for now and left an empty branch, maybe we find use case for it in the future.